project-anemone/routes/web.php

```diff
@@ -25,3 +24,9 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::group([SearchController::class],function(){
+    Route::get('search', [SearchController::class, 'index']);
+    Route::get('autocomplete', [SearchController::class, 'autocomplete'])->name('autocomplete');
+    
+});
```

Comment:
This should probably be a part of the `api.php` as at the moment on the EBP server any of the API calls are only accessible by https://anemone.avalonbloom.com/api/. All calls to https://anemone.avalonbloom.com/ are routed to the React App. The other thing that may need to be decided on is whether a user should be authenticated in order to search. My vote is that you would.

Reply:
fair point! I have moved the endpoints behind API.